nimbus/src/lib.rs

```diff
@@ -63,7 +63,13 @@ impl NimbusClient {
     }
 
     pub fn get_experiment_branch(&self, _slug: String) -> Option<String> {
```

Comment:
I believe this can also be written as:
+        self.enrolled_experiments
+            .iter()
+            .find(|e| e.slug == slug)
+            .map(|e| e.branch_slug.clone())

which seems slighly nicer than the final match

Reply:
Ahh, thanks!  I knew that there must be some more elegant way of writing that bit.